src/component/loginForm.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { login } from "../actions/userAction";
+import "./common/css/main.scss";
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: ""
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
```

Comment:
Try to use the arrow syntax more so you don't have to bind function anymore in React